qutebrowser/html/no_pdfjs.html

```diff
@@ -91,7 +91,7 @@ <h2>Possible fixes</h2>
             If you have installed a packaged version of qutebrowser, make sure
             the required packages for pdf.js are also installed.
             <br/>
-            The package is named <b>pdfjs</b> on Archlinux (AUR) and
+            The package is named <a href="https://archlinux.org/packages/community/any/pdfjs/"><b>pdfjs</b></a> on Archlinux and
             <b>libjs-pdf</b> on Debian.
```

Comment:
Mind adding a link to that as well while you're at it?

Reply:
But the one the AUR is dead... how do I put a link to it?

Reply:
"libjs-pdf on Debian" isn't related to the AUR though? I mean something like https://packages.debian.org/bullseye/libjs-pdf - if you want to link to the Archlinux package, let's link to the Debian package in the same sentence as well...